dataactvalidator/filestreaming/loaderUtils.py

```diff
@@ -72,7 +73,7 @@ def loadCsv(cls,filename,model,interface,fieldMap,fieldOptions):
                     options = fieldOptions[field]
                     if "pad_to_length" in options:
                         padLength = options["pad_to_length"]
-                        row[field] = cls.padToLength(row[field],padLength)
+                        row[field] = Validator.padToLength(row[field],padLength)
```

Comment:
I'm curious about the padToLength function. I see that it also functions as check on the length of incoming data, but couldn't it use Python's zfill string function to do the actual 0 padding?